src/be_db_reward.erl

```diff
@@ -71,17 +77,68 @@ load_block(Conn, _Hash, Block, _Sync, _Ledger, State = #state{}) ->
     {ok, State}.
 
 -type reward_map() :: #{
-    {Account :: libp2p_crypto:pubkey_bin(), Gateway :: libp2p_crypto:pubkey_bin()} =>
+    {Account :: libp2p_crypto:pubkey_bin(), Gateway :: libp2p_crypto:pubkey_bin() | undefined} =>
```

Comment:
Can we add a comment here to specify why `undefined` is now also expected?

Reply:
Ah it was just mis-specced.. it already supported undefined gateways before